net/netutil/udp_gro_forwarding_linux.go

```diff
@@ -0,0 +1,41 @@
+package netutil
+
+import (
+	"fmt"
+
+	"github.com/safchain/ethtool"
+)
+
+func CheckUDPGROForwarding(defaultName, tunName string) (warn, err error) {
+	const txFeature = "tx-udp-segmentation"
+	const rxWantFeature = "rx-udp-gro-forwarding"
+	const rxDoNotWantFeature = "rx-gro-list"
+	const kbLink = "\nSee https://tailscale.com/TODO" // TODO(jwhited): update with real link
```

Comment:
Reminder to update link to doc.

Reply:
Updated in https://github.com/tailscale/tailscale/pull/10071/commits/20c6b407669098a7ee566f377de26b458a64f63c.